sdk/keyvault/azure-security-keyvault-certificates/src/test/java/com/azure/security/keyvault/certificates/CertificateClientTestBase.java

```diff
@@ -419,24 +413,17 @@ void importPemCertificateRunner(Consumer<ImportCertificateOptions> testRunner) t
         testRunner.accept(importCertificateOptions);
     }
 
+    @Test
+    public abstract  void mergeCertificateNotFound(HttpClient httpClient, CertificateServiceVersion serviceVersion);
+
     private byte[] readCertificate(String certName) throws IOException {
         String pemPath = getClass().getClassLoader().getResource(certName).getPath();
-        String pemCert = "";
-        BufferedReader br = new BufferedReader(new FileReader(pemPath));
-        try {
-            String line;
-            while ((line = br.readLine()) != null) {
-                pemCert += line + "\n";
-            }
-        } finally {
-            br.close();
+        if (pemPath.contains(":")) {
+            pemPath = pemPath.substring(1);
```

Comment:
Why was this change required?

Reply:
Not merge the most recent. 
Changed to master one.